src/test/java/pl/javastart/euler2/FibonacciCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+package pl.javastart.euler2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class FibonacciCalculatorTest {
+
+    FibonacciCalculator fibonacciCalculator = new FibonacciCalculator();
+
+    int fibonacciSequenceSum;
+
+    @Test
+    void shouldReturn0WhenGiven0() {
+
+        fibonacciSequenceSum = fibonacciCalculator.getFibonacciSequence(0);
+
+        assertThat(0).isEqualTo(fibonacciSequenceSum);
+
+    }
+
+    @Test
+    void shouldReturn10WhenGiven10() {
+
+        fibonacciSequenceSum = fibonacciCalculator.getFibonacciSequence(10);
+
+        assertThat(10).isEqualTo(fibonacciSequenceSum);
+
+    }
+
+    @Test
+    void shouldReturn44WhenGiven34() {
+
+        fibonacciSequenceSum = fibonacciCalculator.getFibonacciSequence(34);
+
+        assertThat(44).isEqualTo(fibonacciSequenceSum);
```

Comment:
- Równiez odwrotna kolejność
- Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu
- poprawny wynik to raczej 10, bo metoda powinna brać liczby MNIEJSZE od zadanej (a nie mniejsze/równe)

Reply:
Ten test zrobiłem na podstawie treści zadania:
Suma dla elementów parzystych, których wartość nie przekracza 34 to: 2 + 8 + 34 = 44 (1, 3, 5, 13, 21 pomijamy, bo nie są parzyste)

Reply:
Faktycznie, mój błąd! 💪